docker-compose.yml

```diff
@@ -47,14 +45,14 @@ services:
   blackbox:
     image: prom/blackbox-exporter
     network_mode: bridge
+    ports:
+      - "9115:9115"
```

Comment:
blackbox_exporter is typically not accessed outside of the cluster, so normally the ports wouldn't need to be exported.

Reply:
I don't have a strong opinion on this point. I found it useful to have access to the page, mainly during testing, this can be removed.